codalab/lib/docker_util.py

```diff
@@ -0,0 +1,37 @@
+"""
+docker_util.py
+
+Utility class wrapping the docker executable
+"""
+from argcomplete import warn
+import re
+from subprocess import Popen, PIPE
+
+
+class Docker(object):
+    """
+    Wrapper class. Various class methods defined in here serve as wrappers for functionality that calls out to
+    the `docker` binary installed on the host.
+    """
+
+    """
+    Compiled regular expression to parse the stdout of `docker search` calls.
+
+    This captures the beginning of the line up until the first whitespace character, i.e.
+    the first column of the output, corresponding to the image tag.
+    """
+    DOCKER_SEARCH_TAG_REGEX = re.compile(r'^(?P<tag>\S+)\s+')
+
+    @classmethod
+    def search(cls, term, failure_cb=None):
```

Comment:
nit: rename 'term' to 'keyword' to be consistent with name of such variables in bundle_model.py